docs/self-managed/setup/deploy/local/c8run.md

```diff
@@ -0,0 +1,90 @@
+---
+id: c8run
+title: "Local installation with C8Run"
```

Comment:
Can we use Camunda 8 Run (C8Run). C8Run looks like an abbreviation. It will also help with SEO.

Reply:
Late to the party 💃 but here I am!! 😆 

> Can we use Camunda 8 Run (C8Run). C8Run looks like an abbreviation. It will also help with SEO.

@conceptualshark you can consider using `keywords` in the page metadata for things like this.

```
keywords: ["C8Run"]
```